src/Pact/Analyze/Patterns.hs

```diff
@@ -9,7 +9,7 @@ import           Data.Maybe           (isJust)
 import           Data.Text            (Text)
 
 import qualified Pact.Types.Lang      as Lang
-import           Pact.Types.Runtime   (BindType (BindLet, BindSchema),
+import           Pact.Types.Runtime   (BindType (BindLet, BindSchema, BindInlinedCallArgs),
```

Comment:
Can I ask why you have settled on importing individual constructors? Would much prefer `BindType(..)` for code maintainability, but open to debate.

Reply:
I tend to explicitly import everything so that it's clear where things come from, so someone working on a single file doesn't have to grep the entire codebase to figure out what is going on when they encounter a name they're not familiar with. I'm fine with using ellipses for constructors of a type, but I'm not a big fan of polluting a namespace with completely open imports (e.g. `import Control.Lens`).

I've been starting to move towards heavier use of qualified imports for non-infix and uncommon operators. This gives you reduced maintenance burden but at the same time you can understand where things come from without having to look everything up.

For similar reasons, I prefer explicitly bringing fields into scope using `NamedFieldPuns` rather than doing so implicitly with `RecordWildCards`. e.g. `FDefun {_fType,_fArgs,_fBody}` vs `FDefun {..}`